### ISSUE TITLE
Licence Checklist Discussion

### ISSUE THREAD

**Issue by shivajena** [UBC-MDS/tweetlytics#65]

**Comment by mahsasarafrazi** [comment]
Examine the license for your project and consider whether this is the choice you want to make, or whether you want to change the license. Discuss and reason the license choice by opening issues in both Python and R repositories. As it is likely to be a very similar discussion for both projects, one of these issues can just link to the other issue where it is thoroughly discussed

**Comment by mahsasarafrazi** [comment]
We are currently using the MIT License in both our R and Python package. In my opinion, I think the MIT License works best. For future use, our functions could be modified for other applications.
Someday in the future, we might come back to this code to use for later packages or codes and so keeping it simple and being open and permissive such as MIT License sounds good to me!

**Comment by shivajena** [comment]
I agree with MIT License as it supports our idea of the package being open source and being open for contributions, as well as its simple.

**Comment by Amirshoja** [comment]
I agree too, the MIT license supports our goal of creating a ready, easy to use package solution, which would allow developers and researchers, to create applications based on tweet analysis efficiently.
